pkg/sql/parser/sql.y

```diff
@@ -1708,7 +1709,7 @@ stmt_without_legacy_transaction:
 alter_stmt:
   alter_ddl_stmt      // help texts in sub-rule
 | alter_role_stmt     // EXTEND WITH HELP: ALTER ROLE
-| alter_tenant_csetting_stmt  // EXTEND WITH HELP: ALTER TENANT
+| alter_tenant_stmt   /* SKIP DOC */
```

Comment:
why SKIP? I thought we decided to keep all of these? here and below.

Reply:
I thought we were starting with `SKIP DOC` but don't skip the help text?

Reply:
I think we don't skip for create/drop/show tenant now..

Reply:
Yup i have this PR I need to cleanup - https://github.com/cockroachdb/cockroach/pull/91995, let me go do that.